scripts/docker/CC-Devel/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:16.04
+
+RUN \
+    apt-get update \
+    && apt-get install --yes \
```

Comment:
Also, it would be nice if the package list is sorted alphabetically. Makes it easier to read and maintain.

Reply:
Done